tools/keycloak_setup/collection/roles/setup/tasks/create_cli_client.yml

```diff
@@ -0,0 +1,23 @@
+---
+- community.general.keycloak_client:
+    state: present
+    realm: "{{ realm_name }}"
+    name: "{{ client_name }}-cli"
+    client_id: "{{ client_name }}-cli"
+    protocol: openid-connect
+    enabled: true
+    description: "CLI client for Pinakes"
+    validate_certs: "{{ verify_keycloak_ssl }}"
+    admin_url: "{{ admin_url }}"
+    auth_client_id: "{{ auth_client_id }}"
+    auth_keycloak_url: "{{ auth_keycloak_url }}"
+    auth_password: "{{ auth_password }}"
+    auth_realm: "{{ auth_realm }}"
+    auth_username: "{{ auth_username }}"
+    direct_access_grants_enabled: true
+    public_client: false
```

Comment:
@cutwater Should this be a public client?

Reply:
Actually yes, I think it can be a public client.